virttest/storage.py

```diff
@@ -227,6 +229,41 @@ def cmp(x, y):
     return image_filename
 
 
+def copy_nfs_image(params, image_name, root_dir):
+    """
+    copy image from image_path to nfs mount dir if image is not available
+    or corrupted.
+
+    :param params: Test dict params
+    :param image_name: Master image name.
+    :param root_dir: Base directory for relative filenames.
+    :raise: TestSetupFail if image is unavailable/corrupted
+    """
+    nfs_mount_path = params.get("nfs_mount_dir")
+    image_format = params.get("image_format", "qcow2")
+    if nfs_mount_path:
```

Comment:
`nfs_mount_path` is checked as condn but not seeing it gets used anywhere in this method, is it hidden under any other submethods, if then can we move the logic there?

Reply:
yes you are right, we have to use condition with storage_type and setup_local_nfs, I will make the respective change. Thanks for review.